package.json

```diff
@@ -30,43 +32,44 @@
   "dependencies": {
     "async": "~1.4.0",
     "babel": "~6.0.0",
-    "babel-brunch": "~6.0.0",
+    "babel-brunch": "~7.0.0",
     "babel-preset-es2015": "~6.3.13",
     "babel-preset-react": "~6.3.13",
-    "bootstrap": "~3.3.6",
+    "bootstrap": "~4.0.0",
     "bootstrap-solarized": "~1.0.2",
-    "brunch": "~2.9.1",
-    "clean-css-brunch": ">= 1.0 < 1.8",
+    "brunch": "~3.0.0",
+    "clean-css-brunch": ">=3.0.0",
     "cookie-parser": "~1.3.5",
     "cors": "~2.7.1",
     "css-brunch": ">= 1.0 < 1.8",
     "discord.js": "^11.1.0",
-    "express": "~4.15.5",
-    "express-handlebars": "~2.0.1",
+    "express": "~4.16.0",
+    "express-handlebars": "~3.0.0",
     "extend": "~3.0.0",
     "howler": "~1.1.28",
     "javascript-brunch": ">= 1.0 < 1.8",
     "javascript-state-machine": "~2.3.5",
-    "jquery": "~2.2.0",
-    "lodash": "~4.17.16",
+    "jquery": "~3.5.0",
+    "lodash": "~4.17.20",
     "moment": "~2.11.2",
-    "mongoose": "~4.4.6",
-    "morgan": "~1.6.1",
-    "newrelic": "~1.22.1",
+    "mongoose": "~5.7.5",
+    "morgan": "~1.9.1",
+    "newrelic": "~5.13.1",
     "perfect-scrollbar": "~0.6.10",
     "react": "~0.14.6",
     "react-autolink": "~0.2.1",
-    "react-dom": "~0.14.6",
+    "react-dom": "~16.0.1",
```

Comment:
having different versions of react and react-dom may actually be a problem, and needs to be verified on.  

I think it may be causing `'ReactCurrentOwner' of undefined` as unhandled exceptions actually.  

This fix would be to update the react package, but that would also mean handling some refactors with `React.createClass` usage it seems.